src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/respondentsreport/RespondentsReport.java

```diff
@@ -81,19 +81,32 @@ private boolean hasMultipleRespondents(RespondentsReportCaseData caseData) {
     }
 
     private boolean isRepresentativeRepresentingMoreThanOneRespondent(String rep, RespondentsReportCaseData caseData) {
+        if (CollectionUtils.isEmpty(caseData.getRepCollection())) {
+            return false;
+        }
+
+        int count = countRepresentedRespondents(rep, caseData);
+        return count > 1;
+    }
+
+    private int countRepresentedRespondents(String rep, RespondentsReportCaseData caseData) {
         int count = 0;
-        if (CollectionUtils.isNotEmpty(caseData.getRepCollection())) {
-            for (RepresentedTypeRItem repItem : caseData.getRepCollection()) {
-                if (repItem.getValue().getNameOfRepresentative().equals(rep)) {
-                    for (RespondentSumTypeItem respItem : caseData.getRespondentCollection()) {
-                        if (respItem.getValue().getRespondentName().equals(repItem.getValue().getRespRepName())) {
-                            count++;
-                        }
-                    }
-                }
+        for (RepresentedTypeRItem repItem : caseData.getRepCollection()) {
+            if (repItem.getValue().getNameOfRepresentative().equals(rep)) {
+                count += countMatchingRespondents(repItem.getValue().getRespRepName(), caseData);
             }
         }
-        return count > 1;
+        return count;
+    }
+
+    private int countMatchingRespondents(String respRepName, RespondentsReportCaseData caseData) {
+        int count = 0;
+        for (RespondentSumTypeItem respItem : caseData.getRespondentCollection()) {
+            if (respItem.getValue().getRespondentName().equals(respRepName)) {
+                count++;
+            }
+        }
+        return count;
     }
```

Comment:
Using a stream here might make it cleaner, no need to declare `count` as 0 
```suggestion
private int countMatchingRespondents(String respRepName, RespondentsReportCaseData caseData) {
    return (int) caseData.getRespondentCollection().stream()
            .filter(respItem -> respItem.getValue().getRespondentName().equals(respRepName))
            .count();
}
```